packages/cli/templates/react/igr-ts/radial-gauge/default/index.ts

```diff
@@ -12,7 +12,7 @@ class IgrTsRadialGaugeTemplate extends IgniteUIForReactTemplate {
 		this.description = `provides a number of visual elements, like a needle, tick marks, ranges
 							and labels, in order to create a predefined shape and scale.`;
 		// TODO: read version from igniteui-react-core in package.json
-		this.packages = ["igniteui-react-gauges@18.3.0"];
+		this.packages = ["igniteui-react-gauges@~18.6.0"];
```

Comment:
@igdmdimitrov  This can't be right if `igniteui-react-core` is not auto-installed, since `igniteui-react-gauges` 
**requires** it still: https://cdn.jsdelivr.net/npm/igniteui-react-gauges/package.json
Same for charts, grids and pretty much all the react packages I think except `igniteui-react`

Reply:
Nvm, it seems peer deps are also auto-installed actually, so not really needed to be explicitly listed as dep if not useful for the rest of the app, i.e. the core.